### PR TITLE
Revoke admin access

### PR DESCRIPTION
**Pull request by afreekamode**

### CODE REVIEW

**Comment by afreekamode** [comment]
IMG_20191030_074833

**Comment by afreekamode** [comment]
https://gold-firefly-5355.postman.co/collections/7712219-abd4269d-14df-4367-b68e-a6c4974e0ecc?version=latest&workspace=a5ecb11f-6d9e-4f44-87df-f3e8e12b7e3a#018961bd-0368-494b-b240-dc21d604ed09

**Comment by gabrielAndy** [comment]
> 
> 
> https://gold-firefly-5355.postman.co/collections/7712219-abd4269d-14df-4367-b68e-a6c4974e0ecc?version=latest&workspace=a5ecb11f-6d9e-4f44-87df-f3e8e12b7e3a#018961bd-0368-494b-b240-dc21d604ed09

Kindly publish your docs to public

File: app/Http/Controllers/UserProfileController.php [hunk 1]
```diff
@@ -33,6 +33,7 @@ public function all()
         $admins = [];
         $residents = [];
         $gatemans = [];
+<<<<<<< revokeAdminAccess
```
Comment (gabrielAndy) [inline app/Http/Controllers/UserProfileController.php]: Please kindly work on your file again

File: app/User.php [hunk 2]
```diff
@@ -17,7 +17,11 @@ class User extends Authenticatable implements JWTSubject
      * @var array
      */
     protected $fillable = [
+<<<<<<< revokeAdminAccess
```
Comment (gabrielAndy) [inline app/User.php]: This ought to be corrected

File: database/migrations/2019_10_09_002748_create_visitors_table.php [hunk 3]
```diff
@@ -21,16 +21,25 @@ public function up()
             $table->string('phone_no', 25);
             $table->string('purpose', 40);
             $table->string('image', 100)->default('noimage.jpg');
+<<<<<<< revokeAdminAccess
```
Comment (gabrielAndy) [inline database/migrations/2019_10_09_002748_create_visitors_table.php]: This ought to be corrected

File: app/Http/Controllers/UserProfileController.php (hunk 1 above)
Comment (gabrielAndy) [inline app/Http/Controllers/UserProfileController.php]: This ought to be corrected

File: app/User.php (hunk 2 above)
Comment (gabrielAndy) [inline app/User.php]: This ought to be corrected

File: database/migrations/2019_10_09_002748_create_visitors_table.php (hunk 3 above)
Comment (gabrielAndy) [inline database/migrations/2019_10_09_002748_create_visitors_table.php]: This ought to be corrected

File: server.php [hunk 4]
```diff
@@ -1,21 +0,0 @@
-<?php
```
Comment (gabrielAndy) [inline server.php]: I don't know why this file was deleted. Please explain the purpose of deleting this file. Upon then, will I reopen this PR and resolve any issue for merging.

**Comment by gabrielAndy** [comment]
Please kindly update me on why file `server.php` is deleted. Upon then will I re-open the PR for merging.

**Pull request closed without merging**